ygen/codegen.go

```diff
@@ -87,6 +87,18 @@ type GoOpts struct {
 	// produced if the GenerateJSONSchema YANGCodeGenerator field is set to
 	// true.
 	SchemaVarName string
+	// GoyangImportPath specifies the path that should be used in the generated
+	// code for importing the goyang/pkg/yang package. If it is not specified
```

Comment:
ACK --  that was the change that I made here: https://github.com/openconfig/ygot/pull/1/commits/b65a01e2cdbb3bc8cbc928752e8bb31c57ced425#diff-338bea067d2a01b69c90f1c032c6a24b